AutomatedTesting/Gem/PythonTests/atom_renderer/atom_utils/atom_component_helper.py

```diff
@@ -17,3 +23,208 @@
+    ground_plane.components.append(
+        editor.EditorComponentAPIBus(
+            bus.Broadcast, "AddComponentsOfType", ground_plane.id, [mesh_type_id]
+        ).GetValue()[0]
+    )
+    ground_plane_mesh_asset_path = os.path.join("Models", "plane.azmodel")
+    ground_plane_mesh_asset_value = asset.AssetCatalogRequestBus(
+        bus.Broadcast, "GetAssetIdByPath", ground_plane_mesh_asset_path, math.Uuid(), False)
+    ground_plane.get_set_test(1, "Controller|Configuration|Mesh Asset", ground_plane_mesh_asset_value)
+
+    # Create directional_light entity and set the properties
+    directional_light = hydra.Entity("directional_light")
+    directional_light.create_entity(
+        entity_position=math.Vector3(0.0, 0.0, 10.0),
+        components=["Directional Light"],
+        parent_id=default_level.id)
+    directional_light_rotation = math.Vector3(degree_radian_factor * -90.0, 0.0, 0.0)
+    azlmbr.components.TransformBus(
+        azlmbr.bus.Event, "SetLocalRotation", directional_light.id, directional_light_rotation)
+
+    # Create sphere entity and set the properties
+    sphere_entity = hydra.Entity("sphere")
+    sphere_entity.create_entity(
+        entity_position=math.Vector3(0.0, 0.0, 1.0),
+        components=["Material"],
+        parent_id=default_level.id)
+    sphere_material_asset_path = os.path.join("Materials", "Presets", "PBR", "metal_brass_polished.azmaterial")
+    sphere_material_asset_value = asset.AssetCatalogRequestBus(
+        bus.Broadcast, "GetAssetIdByPath", sphere_material_asset_path, math.Uuid(), False)
+    sphere_entity.get_set_test(0, "Default Material|Material Asset", sphere_material_asset_value)
+
+    # Work around to add the correct Atom Mesh component
+    sphere_entity.components.append(
+        editor.EditorComponentAPIBus(
+            bus.Broadcast, "AddComponentsOfType", sphere_entity.id, [mesh_type_id]
+        ).GetValue()[0]
+    )
+    sphere_mesh_asset_path = os.path.join("Models", "sphere.azmodel")
+    sphere_mesh_asset_value = asset.AssetCatalogRequestBus(
+        bus.Broadcast, "GetAssetIdByPath", sphere_mesh_asset_path, math.Uuid(), False)
+    sphere_entity.get_set_test(1, "Controller|Configuration|Mesh Asset", sphere_mesh_asset_value)
+
+    # Create camera component and set the properties
+    camera_entity = hydra.Entity("camera")
+    camera_entity.create_entity(
+        entity_position=math.Vector3(5.5, -12.0, 9.0),
+        components=["Camera"],
+        parent_id=default_level.id)
+    rotation = math.Vector3(
+        degree_radian_factor * -27.0, degree_radian_factor * -12.0, degree_radian_factor * 25.0
+    )
+    azlmbr.components.TransformBus(azlmbr.bus.Event, "SetLocalRotation", camera_entity.id, rotation)
+    camera_entity.get_set_test(0, "Controller|Configuration|Field of view", 60.0)
+    camera.EditorCameraViewRequestBus(azlmbr.bus.Event, "ToggleCameraAsActiveView", camera_entity.id)
+
+
+def attach_component_to_entity(entity_id, component_name):
```

Comment:
nit: consider promoting this to a higher level utility class since it's not really Atom specific.

Reply:
ill cut a ticket for it since it will be out of scope (the hydra tools are in another repo and installed using that repo via CMake)

Reply:
actually i am mistaken and can edit this directly